summaries/sprint5.md

```diff
@@ -0,0 +1,17 @@
+# Week 5 sprint retrospective
+
+## Scrum master (Louis)
+
+## Bettens, Louis
+
+## Jaakik, Marouane
+
+## Konrad, Jonas Franz
+
+## Lachat, Niels Marco
+
+## Lenweiter, Martin
+
+## Vial, Nicolas
+

```

Comment:
```suggestion
## Vial, Nicolas
During this sprint, I modified all the Dataset Management UIs to use the firestore database. Modifying the activities took some work but I didn't encounter any problems. The problems came when rewriting the tests, I had to familiarize myself with firestore and manage to make the tests work regardless of the state of the database used which took me a lot of time but I finally succeeded. I communicated a lot with Jonas during this sprint and we worked hand in hand to implement all the necessary functions for dataset management. I didn't think it would take me so long, so I couldn't start my 2nd issue which was about modifying the UIs to get a good display no matter the screen size. For the rest, everything went well on my side for this sprint.
```